lib/vsc/utils/fancylogger.py

```diff
@@ -343,7 +323,7 @@ def raiseException(self, message, exception=None, catch=False):
             exception = self.RAISE_EXCEPTION_CLASS
 
         self.RAISE_EXCEPTION_LOG_METHOD(fullmessage)
-        raise_with_traceback(exception(message))
+        raise(exception(message)).with_traceback(tb)
```

Comment:
is this correct? `with_traceback` looks an exception method

so probably somethin g like
```
excp = exception(message)
if tb is not None:
    excp = excp.with_traceback(tb)
raise(excp)
```

Reply:
well spotted, I'll fix.